ui/partials/main/thGlobalTopNavPanel.html

```diff
@@ -80,17 +80,60 @@
         </span>
       </span>
 
-      <!-- Global Filter Button -->
-      <span class="btn btn-view-nav btn-right-navbar nav-menu-btn"
-            title="Global job filters"
-            ng-class="{'active': (isFilterPanelShowing)}"
-            ng-click="setFilterPanelShowing(!isFilterPanelShowing)"
```

Comment:
`setFilterPanelShowing()` and `isFilterPanelShowing` in `main.js` are unused after this